ts/WoltLabSuite/Core/Ui/Sortable/List.ts

```diff
@@ -35,52 +57,157 @@ class UiSortableList {
         containerId: "",
         className: "",
         offset: 0,
-        options: {},
+        maxNestingLevel: undefined,
+        toleranceElement: "span",
```

Comment:
This is a breaking change despite the functionality remaining the same. Please add a check after the option initialization if `typeof options.toleranceElement === "string"` and then copy the property over.

That does cause the TS compiler to complain about it but existing JS artifacts remain functional.